icons/test-tube.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M21 6 5.62 21.38c-.83.83-2.17.83-3 0v0a2.12 2.12 0 0 1 0-3L18 3" />
+  <path d="m17 2 5 5" />
+  <path d="M15 12H9" />
```

Comment:
![image](https://user-images.githubusercontent.com/17746067/228534744-f1ceb5ce-d6c7-419b-9281-b41a69309488.png)
These are changes for the thicker version:
```suggestion
  <path d="M21 7 6.82 21.18a2.83 2.83 0 0 1-3.99-.01v0a2.83 2.83 0 0 1 0-4L17 3" />
  <path d="m16 2 6 6" />
  <path d="M16 12H8" />
```

This is the test-tubes variant:
```svg
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M9 2v17.5A2.5 2.5 0 0 1 6.5 22v0A2.5 2.5 0 0 1 4 19.5V2" />
  <path d="M20 2v17.5a2.5 2.5 0 0 1-2.5 2.5v0a2.5 2.5 0 0 1-2.5-2.5V2" />
  <path d="M3 2h7" />
  <path d="M14 2h7" />
  <path d="M9 11H4" />
  <path d="M20 11h-5" />
</svg>
```